main.js

```diff
@@ -1 +1,50 @@
-console.log('hello word');
+let user;
+user = 'John Doe';
+console.log(user);
+
+let student;
+// eslint-disable-next-line prefer-const
+student = 'Daria Ramanowska';
+console.log(student);
+
+user = student; //redefined data to 'Daria Ramanowska'
+console.log(user);
+
+let test = 1;
+test++;
+
+test = test + '1';
+console.log(test); // '21' string
+
+test = test - 1;
+console.log(test); // 20 number
+
+test = Boolean(test === 20);
```

Comment:
What is the magic number 20 for here? Where did you get it?
It doesn't match the task.